build_tools/scripts/bazel_to_cmake.py

```diff
@@ -95,14 +97,30 @@ def _convert_name_block(self, **kwargs):
     #    rule_name
     return "  NAME\n    %s\n" % (kwargs["name"])
 
-  def _convert_namespace_block(self, **kwargs):
+  def _convert_cc_namespace_block(self, **kwargs):
     #  CC_NAMESPACE
     #    "cc_namespace"
     return "  CC_NAMESPACE\n    \"%s\"\n" % (kwargs["cc_namespace"])
 
+  def _convert_cpp_namespace_block(self, **kwargs):
+    #  CPP_NAMESPACE
+    #    "cpp_namespace"
+    return "  CPP_NAMESPACE\n    \"%s\"\n" % (kwargs["cpp_namespace"])
+
   def _convert_translation_block(self, **kwargs):
     return "  TRANSLATION\n    \"%s\"\n" % (kwargs["translation"])
 
+  def _convert_translate_tool_block(self, **kwargs):
+    translate_tool = kwargs.get("translate_tool")
+    print(translate_tool)
+    if translate_tool:
+      translate_tool = translate_tool.replace("//", "")  # iree/base:api
```

Comment:
Can you put the same
```
      # Bazel `//iree/base`     -> CMake `iree::base`
      # Bazel `//iree/base:api` -> CMake `iree::base::api`
```
comment before these lines, or refactor such that the block below and this block call into a helper function that does the substitutions?

If we were to write unit tests for this script, that helper function could then be tested on its own.

Reply:
Didn't hast pytests in mind. Just added a comment for now, will refactor/implement some helper functions next week anyway :)

Reply:
I don't have specific plans [yet] to write unit tests for this script, though I do find it helpful to design code to be easy to test. If it's hard to write tests for a function, it's probably hard to use that function in regular program flow also :)

Reply:
FTR, I think we should be able to roll this target conversion logic into bazel_to_cmake_targets.py

Reply:
FTR I think we should roll all the standard "turn bazel target into cmake target" into a function separate from filtering out what should be in each block.